src/utils/i18nCh.ts

```diff
@@ -1,47 +1,49 @@
 import i18next from "i18next"
 import Backend from "i18next-fs-backend"
-import path    from "path";
+import path from "path";
 import fs, { readdirSync, lstatSync } from "fs"
 import log from "./log";
 import { NODE_MODULES_LOCALES_PATH } from "../constants/constants";
 
 export default class i18nCh {
-
    private static instance: i18nCh
    private i18nInst
    private localesFolder
+   private nodeModulesFolder
    private nameSpaces: string[] = [];
 
-   //_______________________________________________________________________________________________
    private constructor(localesFolder = "", nameSpaces: string[] = [], lang = "en") {
       try {
          if (!localesFolder) {
             // no point to translate these errors as if they should ever happen they are input for devs, not customers
             throw new Error("i18nCh initialization error: path to locales must be provided")
          }
          this.localesFolder = localesFolder
-         if ( nameSpaces.length === 0) {
-            nameSpaces =  this.loadAllNamespaces();
+         this.nodeModulesFolder = NODE_MODULES_LOCALES_PATH
+
+         if (nameSpaces.length === 0) {
+            nameSpaces = this.loadAllNamespaces();
          }
          this.nameSpaces = nameSpaces
 
          this.i18nInst = i18next
          this.i18nInst
-            .use(Backend)
-            .init({
+         .use(Backend)
+         .init({
             initImmediate: false, // false = will load the resources synchronously
             ns: nameSpaces,
             partialBundledLanguages: true,
             lng: lang,
             fallbackLng: "en",
-            preload: readdirSync(localesFolder).filter((fileName) => {
-               const joinedPath = path.join(localesFolder, fileName)
-               return lstatSync(joinedPath).isDirectory()
-            }),
+            preload: this.getLanguageFolders(),
             backend: {
-               loadPath: path.join(localesFolder, `{{lng}}/{{ns}}.json`)
+               loadPath: (lng: any, ns: any) => {
```

Comment:
Is `any` required instead of `string` ?

Reply:
Thanks, I've addressed this in https://github.com/companieshouse/ch-node-utils/pull/15/commits/635819530c81ae48886f75f02db5fb92a6bf423b